circle.yml

```diff
@@ -43,6 +43,19 @@ defaults:
       name: "Restore dependencies cache"
       key: *deps-cache-key
 
+  test-ewasm: &test-ewasm
+    run:
+      name: "Run ewasm state tests"
+      working_directory: ~/project
+      command: |
+        echo "print working directory:"
+        pwd
+        echo "list files in working directory:"
+        ls -al .
+        echo "run tests script."
+        ./scripts/tests.sh
```

Comment:
Seeing as this script clones and rebuilds cpp-ethereum each time, it will be very slow to run tests without using the dependency cache. I would suggest moving the client build step out of the script and into a separate CI job where we can add the caching.

Reply:
UPDATE:
tried doing this. I think it would be too messy without setting up a CMake dependency for cpp-ethereum, or a submodule.